manifests/services.pp

```diff
@@ -20,15 +20,24 @@
     mode   => '0664',
   }
 
-  service { 'usaged':
+  # FROM https://jira.ncsa.illinois.edu/browse/SVC-4552
+  #   usaged only needs to run on idds-prod
+  #   since the usage script is configured to run sockets to idds-prod
+  if ($facts['hostname'] == 'idds-prod') {
```

Comment:
Yuck. Note to fix this logic down the road so we don't have control flow based on hostname.

Reply:
There is already similar logic (basically the opposite condition) in the `profile_idds::users` class. :(